adoptopenjdk-frontend-parent/adoptium-frontend-assets/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>net.adoptopenjdk.api</groupId>
+        <artifactId>adoptopenjdk-frontend-parent</artifactId>
```

Comment:
folder says adoptium-frontend-assets but group id is adoptopenjdk-frontend-parents

Reply:
that is the parent id